build.sh

```
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
+
```

@mgsdk I put this back, whilst we can explicitly throw errors it's nice also not to silently fail